src/performance_counters/register_counters.cpp

```diff
@@ -41,7 +43,14 @@ namespace phylanx { namespace performance_counters
             value.status_ = hpx::performance_counters::status_new_data;
             value.count_ = ++invocation_count_;
 
-            /*value.values_ = std::move(result);*/
+            std::vector<std::int64_t> result;
+            // Extract the values from instances_
+            for (auto const& instance : instances_)
+            {
+                result.push_back(instance->get_duration());
```

Comment:
This counter type is used for both, the timing and the count. That means that it will not return the proper results for the counts.

Reply:
Resolved.